### ISSUE TITLE
sdwan_traffic_data_policy_definition sequences are not accepted.

### ISSUE THREAD

**Issue by andreasbaekdahl** [CiscoDevNet/terraform-provider-sdwan#106]
Sequences for data policy does not appear in the GUI. If you create a new sequence of the policy manually in the GUI and then delete it again, the sequences appear, but without nat_parameter settings. See attached GUI screenshots step 1 - 4 .

```
resource "sdwan_data_ipv4_prefix_list_policy_object" "DIA_EXCLUDE" {
  name = "TF-DIA-EXCLUDE"
  entries = [for prefix in ["10.0.0.0/8", "172.16.0.0/12", "192.168.0.0/16"] :
    {
      prefix = prefix
    }
  ]
}

resource "sdwan_traffic_data_policy_definition" "DIA" {
  name           = "TF-DIA"
  description    = "Direct internet access"
  default_action = "accept"
  sequences = [
    {
      id      = 1
      name    = "Custom"
      type    = "data"
      ip_type = "ipv4"
      match_entries = [
        {
          type                            = "destinationDataPrefixList"
          destination_data_prefix_list_id = sdwan_data_ipv4_prefix_list_policy_object.DIA_EXCLUDE.id
        }
      ]
      action_entries =[]
      base_action = "accept"
    },
    {
      id            = 11
      name          = "Custom"
      type          = "data"
      ip_type       = "ipv4"
      base_action = "accept"
      match_entries =[]
      action_entries = [
        {
          "nat_parameters": [
            {
              "type": "useVpn",
              "vpn_id": 0
            },
            {
              "fallback": true,
              "type": "fallback"
            }
          ],
          "type": "nat"
        }
      ]
    }
  ]
}
```

Missing Terraform-configured sequences:
<img width="1198" alt="GUI step 1" src="https://github.com/CiscoDevNet/terraform-provider-sdwan/assets/26814145/f8a3a018-dcbe-4f55-a22d-9432e96b3fee">

Create new empty sequence manually:
<img width="1206" alt="GUI step 2" src="https://github.com/CiscoDevNet/terraform-provider-sdwan/assets/26814145/28ceb247-5faf-451f-a3af-f7df64ffbc9b">

Delete manually created sequence:
<img width="1211" alt="GUI step 3" src="https://github.com/CiscoDevNet/terraform-provider-sdwan/assets/26814145/4b43356d-2d0e-4e11-9d13-db5892d06b84">

Terraform-configured sequences are now visible, but without nat parameters:
<img width="1204" alt="GUI step 4" src="https://github.com/CiscoDevNet/terraform-provider-sdwan/assets/26814145/1b5d5583-493c-4db1-937c-802eaac96649">

**Comment by danischm** [comment]
It looks like the UI has an issue if the `match` element is missing from a sequence.

**Comment by danischm** [comment]
Yeah, that is what I meant. If the empty list element (`actions` or `match.entries`) is not included the GUI does not render it correctly.

> It does not make any difference wether I specify match_entries = null or match_entries  = [] for the TF resource.
It does not make any difference wether I specify action_entries = null or action_entries = [] for the TF resource.

This does not make a difference because the rendered payload is the same.

This commit should fix the issue: https://github.com/CiscoDevNet/terraform-provider-sdwan/commit/a763bb7544a1378694043912e4796c815d4ae3fe

**Comment by danischm** [comment]
Fixed in v0.2.11 release.
